### PR TITLE
[FIX] GET trades history for addresses without trades

### PR DESCRIPTION
**Pull request by vnavascues**
An easy one, if an address has `db_swaps` proceed with `swaps_to_trades()`. Problem was in this line for empty swaps list:
```python
    @staticmethod
    def swaps_to_trades(swaps: List[AMMSwap]) -> List[AMMTrade]:
        trades = []
        # sort by timestamp and then by log index
        swaps.sort(key=lambda trade: (trade.timestamp, -trade.log_index), reverse=True)
        last_tx_hash = swaps[0].tx_hash  # THIS
        current_swaps: List[AMMSwap] = []

```

### CODE REVIEW

File: rotkehlchen/chain/ethereum/uniswap/uniswap.py [hunk 1]
```diff
@@ -392,8 +392,8 @@ def _get_trades(
                 location=Location.UNISWAP,
                 address=address,
             )
-            db_trades = self.swaps_to_trades(db_swaps)
-            if db_trades:
+            if db_swaps:
+                db_trades = self.swaps_to_trades(db_swaps)
```
Comment (LefterisJP) [inline rotkehlchen/chain/ethereum/uniswap/uniswap.py:396]: @vnavascues Thanks for finding the bug! Can you though solve it by adding the check inside `swaps_to_trades()`?

Also perhaps unit test that function with an empty list?

**Comment by LefterisJP** [comment]
Also don't worry about the failed test. Usual kraken problems when everyone wants to log in when the price is surging :rocket:

File: rotkehlchen/chain/ethereum/uniswap/uniswap.py (hunk 1 above)
Comment (vnavascues) [reply rotkehlchen/chain/ethereum/uniswap/uniswap.py:396]: I will. I didn't want to mess up this function, but better have it there as well.

Comment (LefterisJP) [reply rotkehlchen/chain/ethereum/uniswap/uniswap.py:396]: The function is messed up already :smile: 

It takes a list and returns a list, but if the list it takes has length 0 it raises an exception. The bug that you found is in the function :wink:

File: rotkehlchen/tests/unit/uniswap/test_swaps_to_trades.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+def test_empty_list(uniswap_module):
```
Comment (vnavascues) [inline rotkehlchen/tests/unit/uniswap/test_swaps_to_trades.py:1]: @LefterisJP responded :D

Comment (LefterisJP) [reply rotkehlchen/tests/unit/uniswap/test_swaps_to_trades.py:1]: thanks

**Review by LefterisJP — APPROVED** [review]
thank you. LGTM
